lib/jekyll.rb

```diff
@@ -78,7 +78,7 @@ class << self
     # images and allows you to skip that when working in development.
 
     def env
-      ENV["JEKYLL_ENV"] || "development"
+      ENV["JEKYLL_ENV"] || "local"
```

Comment:
I used examples from the Ruby world, but `development` and `production` are two terms that I would expect to be nearly ubiquitous amongst our userbase, which was the original point that I made poorly before.